testbed/tests_unstable_exe/log_test.go

```diff
@@ -31,4 +34,42 @@ func TestMain(m *testing.M) {
 	testbed.DoTestMain(m, contribPerfResultsSummary)
 }
 
-// This file is left with no tests as a placeholder for future unstable exe tests
+func TestLog10kDPS(t *testing.T) {
```

Comment:
Can we move this to the stable log_test.go instead?

Reply:
sure, I was not sure what exactly `unstable` means

Reply:
The `tests_unstable_exe/log_test.go` is for testing the unstable executable.
Stanza/filelog receiver previously was part of unstable executable. It was moved to `stable` exe yesterday.